src/qri/userProfile.ts

```diff
@@ -15,37 +15,53 @@ export interface UserProfile {
   photo: string
   poster: string
   twitter: string
-  PeerIDs: string[]
-  NetworkAddrs: string[]
+  peerIDs: string[]
+  networkAddrs: string[]
   id: string
   currentKey: string
-  EmailConfirmed: boolean
+  emailConfirmed: boolean
   isAdmin: boolean
 }
 
-export const NewUserProfile = () => {
+export const NewUserProfile = (d: Record<string, any>): UserProfile => {
+  let created = 0
+  let updated = 0
+
+  if (typeof d.created === 'string') {
+    const c = new Date(created)
+    created = c.getTime() / 1000
+  } else if (d.created) {
+    created = d.created
+  }
+
+  if (typeof d.updated === 'string') {
+    const u = new Date(updated)
+    updated = u.getTime() / 1000
+  } else if (d.updated) {
+    updated = d.updated
+  }
```

Comment:
the `UserProfile` coming back from cloud has the updated and created fields as numbers, backend has them coming back as strings. This unifies!